src/containers/my-cooperations/my-cooperations-details/MyCooperationsDetails.styles.ts

```diff
@@ -0,0 +1,71 @@
+import palette from '~/styles/app-theme/app.pallete'
+import { TypographyVariantEnum } from '~/types'
+import { ellipsisTextStyle } from '~/utils/helper-functions'
+
+export const style = {
+  header: {
+    typography: TypographyVariantEnum.H6,
+    color: palette.basic.blueGray,
+    ml: '20px',
+    mb: '30px'
+  },
+  titles: {
+    color: palette.basic.blueGray
+  },
+  title: {
+    fontWeight: '600',
+    ...ellipsisTextStyle(1)
+  },
+  container: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(2, max-content)',
+    gap: '40px',
+    padding: '20px'
+  },
+  profileContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: '10px',
+    pt: '0px',
+    pl: '0px',
```

Comment:
you have `padding`, `pt`, `pl`
try to write it in one property `p`

Reply:
done